components/collective-navbar/NavBarCategoryDropdown.js

```diff
@@ -149,9 +150,7 @@ const CategoryDropdown = styled(Dropdown)`
 
 const getLinkProps = (useAnchor, collective, category) => {
   const anchor = `#category-${category}`;
-  if (useAnchor) {
-    return { href: anchor };
-  } else {
```

Comment:
I don't understand why we're removing this. Anyway, the markup now has nested `<a>` which is invalid HTML

Reply:
![image](https://user-images.githubusercontent.com/1556356/151134753-8420c5d6-051e-40f7-bdf4-fa324001d670.png)

Reply:
I tried removing this to simplify; but I missed the invalid html problem. I've refactored the code to remove the entire `getLinkProps` function and made sure that we don't have the nested `a` tag anymore.